docs/extensibility.md

```diff
@@ -31,8 +31,11 @@ the Agent, which hosts the `Registration` service defined in the gRPC interface.
 ### Use `cargo generate` to clone the Discovery Handler template
 Pull down the [Discovery Handler template](https://github.com/kate-goldenring/akri-discovery-handler-template) using
```

Comment:
these two sentences are redundant. Can you delete the first.

Reply:
Do you mean the h3 heading or the sentence starting with _Pull down ..._?